README.md

```diff
@@ -14,7 +14,7 @@
 | :---        |    :----:   | 
 | Polkadot      | [v0.9.38](https://github.com/paritytech/polkadot/releases/tag/v0.9.38)       |
 | OAK-blockchain   | [v1.9.0](https://github.com/OAK-Foundation/OAK-blockchain/releases/tag/v1.9.0)     |
```

Comment:
This line needs to change too because you are testing with 2.0.0.

Please refer to, https://github.com/OAK-Foundation/xcm-demo/pull/74/files

Reply:
Okay, Let me modify it to https://github.com/OAK-Foundation/OAK-blockchain/releases/tag/v2.0.0